core/src/main/java/io/kestra/core/schedulers/AbstractScheduler.java

```diff
@@ -426,7 +431,12 @@ private void handle() {
                             .labels(f.getFlow().getLabels())
                             .state(new State().withState(State.Type.FAILED))
                             .build();
-                        ZonedDateTime nextExecutionDate = f.getPollingTrigger().nextEvaluationDate();
+                        ZonedDateTime nextExecutionDate = null;
+                        try {
+                            nextExecutionDate = f.getPollingTrigger().nextEvaluationDate(f.getConditionContext(), Optional.of(f.getTriggerContext()));
+                        } catch (Exception e) {
+                            logError(f, e);
+                        }
```

Comment:
This change should be revert, if we cannot get the nextDate with conditions, we get the next date without evaluating the condition. This is important to avoid evaluating the condition on each second.